packages/fx-core/src/common/tools.ts

```diff
@@ -466,3 +466,13 @@ export async function askSubscription(
   }
   return ok(resultSub);
 }
+
+// Determine whether feature flag is enabled based on environment variable setting
+export function isFeatureFlagEnabled(featureFlagName: string): boolean {
+  const flag = process.env[featureFlagName];
+  // can enable feature flag by set environment variable value to "1" or "true"
+  if (flag && (flag === "1" || flag.toLowerCase() === "true")) {
+    return true;
+  }
+  return false;
+}
```

Comment:
Seems can be simplified to 
```typescript
return (flag && (flag === "1" || flag.toLowerCase() === "true"));
```

Reply:
This statement will also generate undefined result, so cannot return directly.

Reply:
Or
`return (flag && (flag === "1" || flag.toLowerCase() === "true"))? true : false;`

Reply:
let me keep current implementation... it seems not a bad code style, and roughly equal to the one line code